### PR TITLE
fix: Make name/email lookups case-insensitive (#5972)

### PR DESCRIPTION
**Pull request by richsalz**
Use icontains so that looking up name or email is case insensitive Added a test

### CODE REVIEW

**Comment by jennifer-richards** [comment]
Would using `__iexact` instead of `__icontains` fix the issue without opening this up as you described in #6008 ?

**Comment by jennifer-richards** [comment]
Would using `__iexact` instead of `__icontains` fix this without opening up a search like you describe in #6008?

**Comment by richsalz** [comment]
Yes, that would fix the web search, but `ajax_select2_search` still uses `__icontains` so the API would still have the problem. And searching for people by things like `salz` is useful.  (I thought there was an open issue for that, but I can't find it.)

File: ietf/person/views.py [hunk 1]
```diff
@@ -69,11 +69,11 @@ def ajax_select2_search(request, model_name):
 
 
 def profile(request, email_or_name):
-    aliases = Alias.objects.filter(name=email_or_name)
+    aliases = Alias.objects.filter(name__icontains=email_or_name)
```
Comment (rjsparks) [inline ietf/person/views.py:72]: how about iexact instead of icontains in both places?

Comment (richsalz) [reply ietf/person/views.py]: added commit to do this.
(It was useful to each just 'salz'  but not worth the #6008 risk)

**Review by rjsparks — APPROVED** [review]

**Pull request merged**